src/browser/web-viewer.js

```diff
@@ -30,7 +37,10 @@ define((require, exports, module) => {
       isRemote: true,
       allowFullScreen: true,
 
-      isVisible: isActive(webViewerCursor) || isSelected(webViewerCursor),
+      isVisible: isActive(webViewerCursor) ||
+                 isSelected(webViewerCursor) ||
+                 !webViewerCursor.get('thumbnail'),
```

Comment:
Do you really have to do that? It doesn't work if you don't set the iframe visible?